README.md

```diff
@@ -44,6 +44,23 @@ Example command to spin up 2 Windows nodes and retain them after test run:
 hack/run-ci-e2e-test.sh -s -k "openshift-dev" -n 2      
 ```
 
+## Running the operator locally
+To run the operator locally (for testing purpose only).
+We need to set up all the environment variables required in [Testing Locally](#testing-locally) as well as 
```

Comment:
I am thinking of opening a PR soon remove `Testing Locally` section as WMCO should run on `hostNetwork`

Reply:
I think you cannot remove the section but rename it to something like `Development workflow`.